ivy/functional/backends/paddle/gradients.py

```diff
@@ -21,8 +21,6 @@
 
 
 def variable(x, /):
-    if ivy.is_int_dtype(x.dtype):
```

Comment:
I'm not sure why we're changing this, this is actually quite useful to avoid code from breaking when someone incorrectly passes an integer array to execute_with_gradients and misses out on specifying that it should be ignored in `xs_grad_idxs`

Reply:
.